frontend/src/components/HomeLoggedOut/HomeLoggedOut.jsx

```diff
@@ -6,10 +6,12 @@ import { useHistory } from "react-router-dom";
 function HomeLoggedOut() {
   const [, setLoggedIn] = useLocalStorage("loggedIn", false);
   const [, setToken] = useLocalStorage("token", null);
+  const [, setDisableAuth] = useLocalStorage("disableAuth", false);
   const history = useHistory();
   axios.get("/auth/login").then(function (response) {
     if (!response.data.enabled) {
```

Comment:
It would be useful to add the `else` part to set the `setDisableAuth` to `false` if the configuration on the server-side has been modified.